tests/at_functional_test/pubspec.yaml

```diff
@@ -20,6 +20,13 @@ dependencies:
   uuid: ^3.0.7
   elliptic: ^0.3.8
 
+dependency_overrides:
+  at_persistence_secondary_server:
+    git:
+      url: https://github.com/atsign-foundation/at_server.git
+      path: packages/at_persistence_secondary_server
+      ref: apkam_sync_persistence_changes
+
```

Comment:
remove dependency override

Reply:
After merging the PR: https://github.com/atsign-foundation/at_server/pull/1619, will replace the dependency_overrides with hosted version.